ds2799/model-ds2799-desc-simple_smdl.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Name": "ds2799_simple",
+  "Description": "simple denoising, see https://nilearn.github.io/stable/modules/generated/nilearn.interfaces.fmriprep.load_confounds_strategy.html",
+  "BIDSModelVersion": "1.0.0",
+  "Input": {
+    "task": [
+      "rest"
+    ],
+    "space": [
+      "MNI152NLin2009cAsym"
+    ]
+  },
+  "Nodes": [
+    {
+      "Level": "Run",
+      "Name": "run",
+      "GroupBy": [
+        "run",
+        "session",
+        "subject"
+      ],
+      "Model": {
+        "Description": "removes effect of 1) realigment parameters, their squares, their derivatives and the derivatives squared and 2) the mean signal measured in CSF, white matter",
```

Comment:
Added some "comments": I believe we should make extensive / intensive use of them in the model zoo.